bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties

```diff
@@ -0,0 +1,399 @@
+channel-type.luxtronikheatpump.WMZ_Heizung.label = Wärmemengenzähler Heizung
+channel-type.luxtronikheatpump.WMZ_Brauchwasser.label = Wärmemengenzähler Brauchwasser
+channel-type.luxtronikheatpump.WMZ_Schwimmbad.label = Wärmemengenzähler Schwimmbad
+channel-type.luxtronikheatpump.WMZ_Seit.label = Wärmemengenzähler Gesamt (seit Reset)
+channel-type.luxtronikheatpump.WMZ_Durchfluss.label = Wärmemengenzähler Durchfluss
+channel-type.luxtronikheatpump.AnalogOut1.label = Analog Ausgang 1
+channel-type.luxtronikheatpump.AnalogOut2.label = Analog Ausgang 2
+channel-type.luxtronikheatpump.Time_Heissgas.label = Sperre zweiter Verdichter Heissgas
+channel-type.luxtronikheatpump.Temp_Lueftung_Zuluft.label = Zulufttemperatur
+channel-type.luxtronikheatpump.Temp_Lueftung_Abluft.label = Ablufttemperatur
+channel-type.luxtronikheatpump.Zaehler_BetrZeitSolar.label = Betriebstundenzähler Solar
+channel-type.luxtronikheatpump.AnalogOut3.label = Analog Ausgang 3
+channel-type.luxtronikheatpump.AnalogOut4.label = Analog Ausgang 4
+channel-type.luxtronikheatpump.Out_VZU.label = Zuluft Ventilator (Abtaufunktion)
+channel-type.luxtronikheatpump.Out_VAB.label = Abluft Ventilator
+channel-type.luxtronikheatpump.Out_VSK.label = Ausgang VSK
+channel-type.luxtronikheatpump.OUT_FRH.label = Ausgang FRH
+channel-type.luxtronikheatpump.AnalogIn2.label = Analog Eingang 2
+channel-type.luxtronikheatpump.AnalogIn3.label = Analog Eingang 3
+channel-type.luxtronikheatpump.SAXin.label = Eingang SAX
+channel-type.luxtronikheatpump.SPLin.label = Eingang SPL
+channel-type.luxtronikheatpump.Compact_exists.label = Lüftungsplatine verbaut
+channel-type.luxtronikheatpump.Durchfluss_WQ.label = Durchfluss Wärmequelle
+channel-type.luxtronikheatpump.LIN_exists.label = LIN BUS verbaut
+channel-type.luxtronikheatpump.LIN_ANSAUG_VERDAMPFER.label = Temperatur Ansaug Verdampfer
+channel-type.luxtronikheatpump.LIN_ANSAUG_VERDICHTER.label = Temperatur Ansaug Verdichter
+channel-type.luxtronikheatpump.LIN_VDH.label = Temperatur Verdichter Heizung
+channel-type.luxtronikheatpump.LIN_UH.label = Überhitzung
+channel-type.luxtronikheatpump.LIN_UH_Soll.label = Überhitzung Soll
+channel-type.luxtronikheatpump.LIN_HD.label = Hochdruck
+channel-type.luxtronikheatpump.LIN_ND.label = Niederdruck
+channel-type.luxtronikheatpump.LIN_VDH_out.label = Ausgang Verdichterheizung
+channel-type.luxtronikheatpump.HZIO_PWM.label = Steuersignal Umwälzpumpe
+channel-type.luxtronikheatpump.HZIO_VEN.label = Ventilator Drehzahl
+# channel-type.luxtronikheatpump.HZIO_EVU2.label = EVU2
+channel-type.luxtronikheatpump.HZIO_STB.label = Sicherheits-Tempeartur-Begrenzer Fussbodenheizung
+channel-type.luxtronikheatpump.SEC_Qh_Soll.label = Leistung Sollwert
+channel-type.luxtronikheatpump.SEC_Qh_Ist.label = Leistung Istwert
+channel-type.luxtronikheatpump.SEC_TVL_Soll.label = Temperatur Vorlauf Soll
+# channel-type.luxtronikheatpump.SEC_Software.label = Software Stand SEC Board
```

Comment:
Can the comments be removed?

Reply:
Could remove them. I had decided to keep them, as they hold the descriptions that are used for those channels in the heatpumps java applet. not sure if those channels might make sense to use at some time....

Reply:
I'd remove them, if they are not used, yet.